src/Calculator/Views/GraphingCalculator/EquationInputArea.xaml.cpp

```diff
@@ -405,8 +414,46 @@ void EquationInputArea::VariableAreaTapped(Object ^ sender, TappedRoutedEventArg
         }
     }
 }
-  
+
 void EquationInputArea::EquationTextBox_EquationFormatRequested(Object ^ sender, MathRichEditBoxFormatRequest ^ e)
 {
     EquationFormatRequested(sender, e);
 }
+
+void EquationInputArea::Slider_ValueChanged(Object ^ sender, RangeBaseValueChangedEventArgs ^ e)
+{
+    //variableSliders = ref new Map<String ^, Delayer ^>();
+    auto slider = static_cast<Slider ^>(sender);
+
+    if (slider->FocusState == Windows::UI::Xaml::FocusState::Unfocused)
```

Comment:
Would be good to add a comment as to why this is here so we don’t delete it later